packages/core/src/variables/VariableRenderer.ts

```diff
@@ -28,6 +34,15 @@ export class VariableRenderer {
     this.nj.on('load', (name, source) => {
       this.pageSources.staticIncludeSrc.push({ to: source.path });
     });
+
+    this.variablesFilePath = fsUtil.ensurePosix(path.join(siteRootPath, USER_VARIABLES_PATH));
+  }
+
+  getModifiedVariablesFileContent(): string {
+    const variablesFileContent = fs.existsSync(this.variablesFilePath)
+      ? fs.readFileSync(this.variablesFilePath, 'utf8') : '';
+
+    return variablesFileContent.replace(VARIABLE_REGEX, NUNJUCKS_COMMENT_SYNTAX);
```

Comment:
@LamJiuFong correct me if I am wrong.
I thinkis trying to comment out the variables so that it will not be rendered as stated earlier. As what he is trying to do is to put nunjunks variables on top of the template then render it. To load the nunjucks variables.

If variables is not commented, then it will be rendered.

Reply:
@EltonGohJH You are right, it is to prevent the variables from being rendered